### PR TITLE
Migrate manual changelog to github-changelog-generator

### LINKED ISSUE #7: Migrate changelog to github-changelog-generator
This is described here: https://github.com/github-changelog-generator/github-changelog-generator#migrating-from-a-manual-changelog

This allows a fully automated changelog generation.

### PR DESCRIPTION
**Pull request by Normo**
```shell
github-changelog-generator -u hifis-net -p ansible-role-haproxy --token <my-github-token> --since-tag v1.5.0
```

* Closes #7

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+- Implement GitHub actions workflows [\#1](https://github.com/hifis-net/ansible-role-haproxy/pull/1) ([Normo](https://github.com/Normo))
 
-Group your changes into these categories:
+<!--
+SPDX-FileCopyrightText: 2020 Helmholtz Centre for Environmental Research (UFZ)
+SPDX-FileCopyrightText: 2020 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
 
-`Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `Security`.
+SPDX-License-Identifier: Apache-2.0
+-->
```
Comment (christianhueserhzdr) [inline CHANGELOG.md:17]: Dear Norman @Normo, as already pointed out, I found it odd to see these copyright texts and license identifier not at the top of the file.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 
-## [1.5.0](https://gitlab.com/hifis/ansible/haproxy-role/-/releases/v1.5.0) - 2022-02-22
+## [v1.5.0](https://github.com/hifis-net/ansible-role-haproxy/releases/tag/v1.5.0) - 2022-02-22
 
 [List of commits](https://gitlab.com/hifis/ansible/haproxy-role/-/compare/v1.4.0...v1.5.0)
```
Comment (christianhueserhzdr) [inline CHANGELOG.md:21]: Dear Norman @Normo, is it possible to tell the tool to exchange these links as well or do we actually want to link to GitLab here?

**Review by christianhueserhzdr — COMMENTED** [review]
Dear Norman @Normo, I have just a few questions regarding the auto-generated CHANGELOG file. Otherwise, I am fine with those changes. Thanks.

File: CHANGELOG.md (hunk 1 above)
Comment (christianhueserhzdr) [reply CHANGELOG.md:17]: Dear Norman @Normo, okay, let's leave the license header where it is.

File: CHANGELOG.md (hunk 2 above)
Comment (christianhueserhzdr) [inline CHANGELOG.md:21]: Dear Norman @Normo, this is also the wrong URL pointing to GitLab and not to GitHub, right? Shall we change that in all places in this CHANGELOG file? Or is it by intention to leave those GitLab URLs in place in this file?

**Review by christianhueserhzdr — CHANGES_REQUESTED** [review]
Dear Norman @Normo, I had the same question as in the other PR in the SSH-Keys repository. Shall we point to GitHub instead of GitLab for the lists of commits?

File: CHANGELOG.md (hunk 2 above)
Comment (Normo) [reply CHANGELOG.md]: I updated the URLs. Please check again @christianhueserhzdr

**Review by christianhueserhzdr — APPROVED** [review]
Dear Norman @Normo, I am all fine with these changes. Thanks. I'll approve and merge them now.

**Pull request merged**